### PR TITLE
Feature/ Redis Cloud promo

### PR DESCRIPTION
**Pull request by ArtyomPodymov**
#RI-2219 - get Redis setup info on UI from resources

### CODE REVIEW

File: redisinsight/ui/src/pages/home/components/AddInstanceControls/AddInstanceControls.tsx [hunk 1]
```diff
@@ -1,33 +1,53 @@
...
 }
 
 const AddInstanceControls = ({ onAddInstance, direction, welcomePage = false }: Props) => {
+  const [promoData, setPromoData] = useState<ContentCreateDBItem>()
+  const [guides, setGuides] = useState<IHelpGuide[]>([])
+  const { loading, data } = useSelector(contentSelector)
+  const { theme } = useContext(ThemeContext)
+
+  useEffect(() => {
+    if (!loading && !isEmpty(data)) {
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/pages/home/components/AddInstanceControls/AddInstanceControls.tsx:35]: use return early here, please

File: redisinsight/ui/src/pages/home/components/HelpLinksMenu/HelpLinksMenu.tsx [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-import { HELP_LINKS, HelpLink } from 'uiSrc/pages/home/constants/help-links'
+
+import { IHelpGuide } from 'uiSrc/pages/home/constants/help-links'
 
 import styles from './styles.module.scss'
 
 interface IProps {
-  onLinkClick: (link: HelpLink) => void;
+  onLinkClick?: (link: string) => void;
+  items: Array<IHelpGuide>
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/pages/home/components/HelpLinksMenu/HelpLinksMenu.tsx:10]: I think it better to use `IHelpGuide[]`

File: redisinsight/ui/src/pages/home/constants/help-links.ts [hunk 3]
```diff
@@ -1,36 +1,30 @@
...
-  BuildRedisFromSource = 'buildRedisFromSource',
-  CreateOnDocker = 'createOnDocker',
-  CreateOnMac = 'createOnMac',
+export interface IHelpGuide {
+  id: string;
+  title: string;
+  url: string;
+  description?: string;
+  event?: string;
+  primary?: boolean;
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/pages/home/constants/help-links.ts:9]: please, remove semicolons from interfaces and props in modified files

File: redisinsight/ui/src/pages/home/components/AddInstanceControls/AddInstanceControls.tsx (hunk 1 above)
Comment (ArtyomPodymov) [reply redisinsight/ui/src/pages/home/components/AddInstanceControls/AddInstanceControls.tsx:35]: where did you find `return` from the function?

Comment (ArtyomPodymov) [reply redisinsight/ui/src/pages/home/components/AddInstanceControls/AddInstanceControls.tsx]: done

**Review by romansergeenkosofteq — DISMISSED** [review]

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**